common/tools/js-mcp-server/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@azure-tools/js-mcp-server",
```

Comment:
do we want to have JS as a suffix instead of the prefix?

Reply:
renamed to `@azure-tools/mcp-server`